spec/bolt/cli_spec.rb

```diff
@@ -89,6 +89,81 @@ def stub_config(file_content = {})
     end
   end
 
+  context 'guide' do
+    let(:guides_path) { File.expand_path(File.join(__dir__, '..', '..', 'guides')) }
+    let(:config)      { double('config', format: nil) }
+
+    let(:guides) do
+      Dir.children(guides_path).sort.each_with_object({}) do |child, acc|
```

Comment:
This is the same logic used in `cli.rb`, so if the logic fails this will still pass. I think you want to hardcode the expected output in this case.